src/app/profile/yourGiving/givingRecipientView/givingRecipientView.tpl.html

```diff
@@ -0,0 +1,2 @@
+<donation-recipient ng-repeat="recipient in $ctrl.recipients | orderBy:'designation-name'"
```

Comment:
There are lots of people with a year to date amount of 0 and no other useful data other than name and designation number. Should this only be showing active/recent donations? This could also be an issue with my old data.